modules/apps/object/object-test/src/testFunctional/tests/ObjectNotifications.testcase

```diff
@@ -41,7 +41,7 @@ definition {
 		}
 
 		task ("And given a notification template is created") {
-			Notifications.notificationTemplateAdmin();
+			Notifications.goToNotificationTemplateAdmin();
```

Comment:
Don't fix update the macro names in a separate commit after applying them, just update them in the original commit.

FYI a general practice at Liferay is to do the following:
- Add a macro in one commit
- Use a second commit to apply the new macro to our tests using the commit message `Apply`
   - You can find examples of this practice if you search the commit history for `Apply`